created tandc and policy components with mobile parts, changed favico…

tanvi0612 opened the pull request:
…n, changed footer to show socials in a row, distribute-mobile half done.

tanvi0612 committed:
created tandc and policy components with mobile parts, changed favicon, changed footer to show socials in a row, distribute-mobile half done

Utkarshbhimte commented on lumierevr2018/package.json:
```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "jquery": "^3.2.1",
```
remove this

Utkarshbhimte commented on lumierevr2018/src/components/footerComponent/footer.jsx:
```diff
@@ -9,16 +11,16 @@ class Footer extends Component {
             <ul className="footer-col">
               <li>
-                <a href="/">Join Us</a>
+                 <Link to="/careers">Join Us</Link>
```
use NavLink instead of Link

Utkarshbhimte commented on lumierevr2018/src/components/footerComponent/footer.jsx:
```diff
@@ -31,10 +33,14 @@ class Footer extends Component {
-              <img src={require("../../images/footer/fb.png")} />
-              <img src={require("../../images/footer/yt.png")} />
+              </div>
+              <div className="footer-row">
```
can you post an image of the difference here?

tanvi0612 replied:
image of the difference?

Utkarshbhimte replied:
like screenshot of what changes you made on the footer

Utkarshbhimte commented on lumierevr2018/src/pages/DistributePage/DistributePage.js:
```diff
@@ -6,7 +6,7 @@ class Distribute extends Component {
   render() {
     return (
       <div className="distribute-page page">
-        <div className="container-fluid">
```
add a div with class "container no-limits" here

Utkarshbhimte commented on lumierevr2018/src/pages/PrivacyPolicyPage/PrivacyPolicyPage.js:
```diff
@@ -0,0 +1,181 @@
+class PrivacyPolicyPage extends Component {
+  render() {
+    return (
+      <div className = "privacypolicypage-container">
```
change it to "policy-page page"

Utkarshbhimte replied:
and also "container no-limits" inside that

Utkarshbhimte commented on lumierevr2018/src/pages/PrivacyPolicyPage/PrivacyPolicyPage.scss:
```diff
@@ -0,0 +1,19 @@
+	font-size: 20px;
+
+	@include mobile-lg {
+      flex-direction: column;
```
why adding all this without ```display: flex```?

Utkarshbhimte commented on lumierevr2018/src/pages/TermsAndConditionsPage/TermsAndConditionsPage.js:
```diff
@@ -0,0 +1,155 @@
+class TermsAndConditionsPage extends Component {
+  render() {
+    return (
+      <div className = "tandcpage-container">
```
```tandc-page page``` class here too and ```container no-limits```

tanvi0612 committed:
Update package.json

jquery dependency removed.

tanvi0612 committed:
Update footer.jsx

Changed <Link>s to <NavLink>s and added Blog ref.

tanvi0612 committed:
Update DistributePage.js

div with 'container no-limits' added

tanvi0612 committed:
Update PrivacyPolicyPage.js

changed class to "policy-page page" and also added "container no-limits" inside that

tanvi0612 committed:
Update PrivacyPolicyPage.scss

class name updated, and flex property added.

tanvi0612 committed:
Update TermsAndConditionsPage.js

tandc-page page class name updated

tanvi0612 committed:
Update TermsAndConditionsPage.scss

tandc-page class name updated

Utkarshbhimte committed:
Merge branch 'master' into privacyPolicy

Utkarshbhimte committed:
Update footer.jsx

Utkarshbhimte committed:
Merge branch 'master' into privacyPolicy
